### DISCUSSION TITLE
Unable to Update or Reinstall on an Intel Mac Pro (2019) with Sonoma 14.4.1

### DISCUSSION THREAD

**Discussion by elstalk** [Homebrew/discussions#5327] [Everyday usage]
### Output of `brew config`

```shell
Last login: Mon Apr 29 16:17:07 on ttys001
elsmith@macncheese ~ % brew config
==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
Already downloaded: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
Error: Checksum mismatch.
Expected: 02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
  Actual: 709a870e862579355398c0115ec4d9995a48daf463ca066d671ca4f40afd9927
 Archive: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
To retry an incomplete download, remove the file above.
Error: Failed to install Homebrew Portable Ruby (and your system version is too old)!
elsmith@macncheese ~ %
```

### Output of `brew doctor`

```shell
elsmith@macncheese ~ % brew doctor
==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
Already downloaded: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
Error: Checksum mismatch.
Expected: 02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
  Actual: 709a870e862579355398c0115ec4d9995a48daf463ca066d671ca4f40afd9927
 Archive: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
To retry an incomplete download, remove the file above.
Error: Failed to install Homebrew Portable Ruby (and your system version is too old)!
elsmith@macncheese ~ %
```

### Description of issue

I tried to update brew with "brew update" and got the checksum errors. I run "brew config" and "brew doctor" and get the exact same checksum errors. I uninstalled and reinstalled - and the reinstall also gives me the same errors:

==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew
==> The following new directories will be created:
/usr/local/include
/usr/local/opt
/usr/local/var/homebrew/linked
/usr/local/Cellar
/usr/local/Caskroom
/usr/local/Frameworks

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /bin/mkdir -p /usr/local/include /usr/local/opt /usr/local/var/homebrew/linked /usr/local/Cellar /usr/local/Caskroom /usr/local/Frameworks
==> /usr/bin/sudo /bin/chmod ug=rwx /usr/local/include /usr/local/opt /usr/local/var/homebrew/linked /usr/local/Cellar /usr/local/Caskroom /usr/local/Frameworks
==> /usr/bin/sudo /usr/sbin/chown elsmith /usr/local/include /usr/local/opt /usr/local/var/homebrew/linked /usr/local/Cellar /usr/local/Caskroom /usr/local/Frameworks
==> /usr/bin/sudo /usr/bin/chgrp admin /usr/local/include /usr/local/opt /usr/local/var/homebrew/linked /usr/local/Cellar /usr/local/Caskroom /usr/local/Frameworks
==> /usr/bin/sudo /bin/mkdir -p /usr/local/Homebrew
==> /usr/bin/sudo /usr/sbin/chown -R elsmith:admin /usr/local/Homebrew
==> /usr/bin/sudo /bin/mkdir -p /Users/elsmith/Library/Caches/Homebrew
==> /usr/bin/sudo /bin/chmod g+rwx /Users/elsmith/Library/Caches/Homebrew
==> /usr/bin/sudo /usr/sbin/chown -R elsmith /Users/elsmith/Library/Caches/Homebrew
==> Downloading and installing Homebrew...
From https://github.com/Homebrew/brew
 * [new branch]      auto_update_tap                                        -> origin/auto_update_tap
 * [new branch]      dependabot/bundler/Library/Homebrew/json_schemer-2.2.1 -> origin/dependabot/bundler/Library/Homebrew/json_schemer-2.2.1
 * [new branch]      deps-filters                                           -> origin/deps-filters
 * [new branch]      gems_change_privilege                                  -> origin/gems_change_privilege
 * [new branch]      load-internal-cask-json-v3                             -> origin/load-internal-cask-json-v3
 * [new branch]      master                                                 -> origin/master
 * [new branch]      neon-proxy-5201                                        -> origin/neon-proxy-5201
 * [new tag]               0.1                                                    -> 0.1
 * [new tag]               0.2                                                    -> 0.2
 * [new tag]               0.3                                                    -> 0.3
 * [new tag]               0.4                                                    -> 0.4
 * [new tag]               0.5                                                    -> 0.5
 * [new tag]               0.6                                                    -> 0.6
 * [new tag]               0.7                                                    -> 0.7
 * [new tag]               0.7.1                                                  -> 0.7.1
 * [new tag]               0.8                                                    -> 0.8
 * [new tag]               0.8.1                                                  -> 0.8.1
 * [new tag]               0.9                                                    -> 0.9
 * [new tag]               0.9.1                                                  -> 0.9.1
 * [new tag]               0.9.2                                                  -> 0.9.2
 * [new tag]               0.9.3                                                  -> 0.9.3
 * [new tag]               0.9.4                                                  -> 0.9.4
 * [new tag]               0.9.5                                                  -> 0.9.5
 * [new tag]               0.9.8                                                  -> 0.9.8
 * [new tag]               0.9.9                                                  -> 0.9.9
 * [new tag]               1.0.0                                                  -> 1.0.0
 * [new tag]               1.0.1                                                  -> 1.0.1
 * [new tag]               1.0.2                                                  -> 1.0.2
 * [new tag]               1.0.3                                                  -> 1.0.3
 * [new tag]               1.0.4                                                  -> 1.0.4
 * [new tag]               1.0.5                                                  -> 1.0.5
 * [new tag]               1.0.6                                                  -> 1.0.6
 * [new tag]               1.0.7                                                  -> 1.0.7
 * [new tag]               1.0.8                                                  -> 1.0.8
 * [new tag]               1.0.9                                                  -> 1.0.9
 * [new tag]               1.1.0                                                  -> 1.1.0
 * [new tag]               1.1.1                                                  -> 1.1.1
 * [new tag]               1.1.10                                                 -> 1.1.10
 * [new tag]               1.1.11                                                 -> 1.1.11
 * [new tag]               1.1.12                                                 -> 1.1.12
 * [new tag]               1.1.13                                                 -> 1.1.13
 * [new tag]               1.1.2                                                  -> 1.1.2
 * [new tag]               1.1.3                                                  -> 1.1.3
 * [new tag]               1.1.4                                                  -> 1.1.4
 * [new tag]               1.1.5                                                  -> 1.1.5
 * [new tag]               1.1.6                                                  -> 1.1.6
 * [new tag]               1.1.7                                                  -> 1.1.7
 * [new tag]               1.1.8                                                  -> 1.1.8
 * [new tag]               1.1.9                                                  -> 1.1.9
 * [new tag]               1.2.0                                                  -> 1.2.0
 * [new tag]               1.2.1                                                  -> 1.2.1
 * [new tag]               1.2.2                                                  -> 1.2.2
 * [new tag]               1.2.3                                                  -> 1.2.3
 * [new tag]               1.2.4                                                  -> 1.2.4
 * [new tag]               1.2.5                                                  -> 1.2.5
 * [new tag]               1.2.6                                                  -> 1.2.6
 * [new tag]               1.3.0                                                  -> 1.3.0
 * [new tag]               1.3.1                                                  -> 1.3.1
 * [new tag]               1.3.2                                                  -> 1.3.2
 * [new tag]               1.3.3                                                  -> 1.3.3
 * [new tag]               1.3.4                                                  -> 1.3.4
 * [new tag]               1.3.5                                                  -> 1.3.5
 * [new tag]               1.3.6                                                  -> 1.3.6
 * [new tag]               1.3.7                                                  -> 1.3.7
 * [new tag]               1.3.8                                                  -> 1.3.8
 * [new tag]               1.3.9                                                  -> 1.3.9
 * [new tag]               1.4.0                                                  -> 1.4.0
 * [new tag]               1.4.1                                                  -> 1.4.1
 * [new tag]               1.4.2                                                  -> 1.4.2
 * [new tag]               1.4.3                                                  -> 1.4.3
 * [new tag]               1.5.0                                                  -> 1.5.0
 * [new tag]               1.5.1                                                  -> 1.5.1
 * [new tag]               1.5.10                                                 -> 1.5.10
 * [new tag]               1.5.11                                                 -> 1.5.11
 * [new tag]               1.5.12                                                 -> 1.5.12
 * [new tag]               1.5.13                                                 -> 1.5.13
 * [new tag]               1.5.14                                                 -> 1.5.14
 * [new tag]               1.5.2                                                  -> 1.5.2
 * [new tag]               1.5.3                                                  -> 1.5.3
 * [new tag]               1.5.4                                                  -> 1.5.4
 * [new tag]               1.5.5                                                  -> 1.5.5
 * [new tag]               1.5.6                                                  -> 1.5.6
 * [new tag]               1.5.7                                                  -> 1.5.7
 * [new tag]               1.5.8                                                  -> 1.5.8
 * [new tag]               1.5.9                                                  -> 1.5.9
 * [new tag]               1.6.0                                                  -> 1.6.0
 * [new tag]               1.6.1                                                  -> 1.6.1
 * [new tag]               1.6.10                                                 -> 1.6.10
 * [new tag]               1.6.11                                                 -> 1.6.11
 * [new tag]               1.6.12                                                 -> 1.6.12
 * [new tag]               1.6.13                                                 -> 1.6.13
 * [new tag]               1.6.14                                                 -> 1.6.14
 * [new tag]               1.6.15                                                 -> 1.6.15
 * [new tag]               1.6.16                                                 -> 1.6.16
 * [new tag]               1.6.17                                                 -> 1.6.17
 * [new tag]               1.6.2                                                  -> 1.6.2
 * [new tag]               1.6.3                                                  -> 1.6.3
 * [new tag]               1.6.4                                                  -> 1.6.4
 * [new tag]               1.6.5                                                  -> 1.6.5
 * [new tag]               1.6.6                                                  -> 1.6.6
 * [new tag]               1.6.7                                                  -> 1.6.7
 * [new tag]               1.6.8                                                  -> 1.6.8
 * [new tag]               1.6.9                                                  -> 1.6.9
 * [new tag]               1.7.0                                                  -> 1.7.0
 * [new tag]               1.7.1                                                  -> 1.7.1
 * [new tag]               1.7.2                                                  -> 1.7.2
 * [new tag]               1.7.3                                                  -> 1.7.3
 * [new tag]               1.7.4                                                  -> 1.7.4
 * [new tag]               1.7.5                                                  -> 1.7.5
 * [new tag]               1.7.6                                                  -> 1.7.6
 * [new tag]               1.7.7                                                  -> 1.7.7
 * [new tag]               1.8.0                                                  -> 1.8.0
 * [new tag]               1.8.1                                                  -> 1.8.1
 * [new tag]               1.8.2                                                  -> 1.8.2
 * [new tag]               1.8.3                                                  -> 1.8.3
 * [new tag]               1.8.4                                                  -> 1.8.4
 * [new tag]               1.8.5                                                  -> 1.8.5
 * [new tag]               1.8.6                                                  -> 1.8.6
 * [new tag]               1.9.0                                                  -> 1.9.0
 * [new tag]               1.9.1                                                  -> 1.9.1
 * [new tag]               1.9.2                                                  -> 1.9.2
 * [new tag]               1.9.3                                                  -> 1.9.3
 * [new tag]               2.0.0                                                  -> 2.0.0
 * [new tag]               2.0.1                                                  -> 2.0.1
 * [new tag]               2.0.2                                                  -> 2.0.2
 * [new tag]               2.0.3                                                  -> 2.0.3
 * [new tag]               2.0.4                                                  -> 2.0.4
 * [new tag]               2.0.5                                                  -> 2.0.5
 * [new tag]               2.0.6                                                  -> 2.0.6
 * [new tag]               2.1.0                                                  -> 2.1.0
 * [new tag]               2.1.1                                                  -> 2.1.1
 * [new tag]               2.1.10                                                 -> 2.1.10
 * [new tag]               2.1.11                                                 -> 2.1.11
 * [new tag]               2.1.12                                                 -> 2.1.12
 * [new tag]               2.1.13                                                 -> 2.1.13
 * [new tag]               2.1.14                                                 -> 2.1.14
 * [new tag]               2.1.15                                                 -> 2.1.15
 * [new tag]               2.1.16                                                 -> 2.1.16
 * [new tag]               2.1.2                                                  -> 2.1.2
 * [new tag]               2.1.3                                                  -> 2.1.3
 * [new tag]               2.1.4                                                  -> 2.1.4
 * [new tag]               2.1.5                                                  -> 2.1.5
 * [new tag]               2.1.6                                                  -> 2.1.6
 * [new tag]               2.1.7                                                  -> 2.1.7
 * [new tag]               2.1.8                                                  -> 2.1.8
 * [new tag]               2.1.9                                                  -> 2.1.9
 * [new tag]               2.2.0                                                  -> 2.2.0
 * [new tag]               2.2.1                                                  -> 2.2.1
 * [new tag]               2.2.10                                                 -> 2.2.10
 * [new tag]               2.2.11                                                 -> 2.2.11
 * [new tag]               2.2.12                                                 -> 2.2.12
 * [new tag]               2.2.13                                                 -> 2.2.13
 * [new tag]               2.2.14                                                 -> 2.2.14
 * [new tag]               2.2.15                                                 -> 2.2.15
 * [new tag]               2.2.16                                                 -> 2.2.16
 * [new tag]               2.2.17                                                 -> 2.2.17
 * [new tag]               2.2.2                                                  -> 2.2.2
 * [new tag]               2.2.3                                                  -> 2.2.3
 * [new tag]               2.2.4                                                  -> 2.2.4
 * [new tag]               2.2.5                                                  -> 2.2.5
 * [new tag]               2.2.6                                                  -> 2.2.6
 * [new tag]               2.2.7                                                  -> 2.2.7
 * [new tag]               2.2.8                                                  -> 2.2.8
 * [new tag]               2.2.9                                                  -> 2.2.9
 * [new tag]               2.3.0                                                  -> 2.3.0
 * [new tag]               2.4.0                                                  -> 2.4.0
 * [new tag]               2.4.1                                                  -> 2.4.1
 * [new tag]               2.4.10                                                 -> 2.4.10
 * [new tag]               2.4.11                                                 -> 2.4.11
 * [new tag]               2.4.12                                                 -> 2.4.12
 * [new tag]               2.4.13                                                 -> 2.4.13
 * [new tag]               2.4.14                                                 -> 2.4.14
 * [new tag]               2.4.15                                                 -> 2.4.15
 * [new tag]               2.4.16                                                 -> 2.4.16
 * [new tag]               2.4.2                                                  -> 2.4.2
 * [new tag]               2.4.3                                                  -> 2.4.3
 * [new tag]               2.4.4                                                  -> 2.4.4
 * [new tag]               2.4.5                                                  -> 2.4.5
 * [new tag]               2.4.6                                                  -> 2.4.6
 * [new tag]               2.4.7                                                  -> 2.4.7
 * [new tag]               2.4.8                                                  -> 2.4.8
 * [new tag]               2.4.9                                                  -> 2.4.9
 * [new tag]               2.5.0                                                  -> 2.5.0
 * [new tag]               2.5.1                                                  -> 2.5.1
 * [new tag]               2.5.10                                                 -> 2.5.10
 * [new tag]               2.5.11                                                 -> 2.5.11
 * [new tag]               2.5.12                                                 -> 2.5.12
 * [new tag]               2.5.2                                                  -> 2.5.2
 * [new tag]               2.5.3                                                  -> 2.5.3
 * [new tag]               2.5.4                                                  -> 2.5.4
 * [new tag]               2.5.5                                                  -> 2.5.5
 * [new tag]               2.5.6                                                  -> 2.5.6
 * [new tag]               2.5.7                                                  -> 2.5.7
 * [new tag]               2.5.8                                                  -> 2.5.8
 * [new tag]               2.5.9                                                  -> 2.5.9
 * [new tag]               2.6.0                                                  -> 2.6.0
 * [new tag]               2.6.1                                                  -> 2.6.1
 * [new tag]               2.6.2                                                  -> 2.6.2
 * [new tag]               2.7.0                                                  -> 2.7.0
 * [new tag]               2.7.1                                                  -> 2.7.1
 * [new tag]               2.7.2                                                  -> 2.7.2
 * [new tag]               2.7.3                                                  -> 2.7.3
 * [new tag]               2.7.4                                                  -> 2.7.4
 * [new tag]               2.7.5                                                  -> 2.7.5
 * [new tag]               2.7.6                                                  -> 2.7.6
 * [new tag]               2.7.7                                                  -> 2.7.7
 * [new tag]               3.0.0                                                  -> 3.0.0
 * [new tag]               3.0.1                                                  -> 3.0.1
 * [new tag]               3.0.10                                                 -> 3.0.10
 * [new tag]               3.0.11                                                 -> 3.0.11
 * [new tag]               3.0.2                                                  -> 3.0.2
 * [new tag]               3.0.3                                                  -> 3.0.3
 * [new tag]               3.0.4                                                  -> 3.0.4
 * [new tag]               3.0.5                                                  -> 3.0.5
 * [new tag]               3.0.6                                                  -> 3.0.6
 * [new tag]               3.0.7                                                  -> 3.0.7
 * [new tag]               3.0.8                                                  -> 3.0.8
 * [new tag]               3.0.9                                                  -> 3.0.9
 * [new tag]               3.1.0                                                  -> 3.1.0
 * [new tag]               3.1.1                                                  -> 3.1.1
 * [new tag]               3.1.10                                                 -> 3.1.10
 * [new tag]               3.1.11                                                 -> 3.1.11
 * [new tag]               3.1.12                                                 -> 3.1.12
 * [new tag]               3.1.2                                                  -> 3.1.2
 * [new tag]               3.1.3                                                  -> 3.1.3
 * [new tag]               3.1.4                                                  -> 3.1.4
 * [new tag]               3.1.5                                                  -> 3.1.5
 * [new tag]               3.1.6                                                  -> 3.1.6
 * [new tag]               3.1.7                                                  -> 3.1.7
 * [new tag]               3.1.8                                                  -> 3.1.8
 * [new tag]               3.1.9                                                  -> 3.1.9
 * [new tag]               3.2.0                                                  -> 3.2.0
 * [new tag]               3.2.1                                                  -> 3.2.1
 * [new tag]               3.2.10                                                 -> 3.2.10
 * [new tag]               3.2.11                                                 -> 3.2.11
 * [new tag]               3.2.12                                                 -> 3.2.12
 * [new tag]               3.2.13                                                 -> 3.2.13
 * [new tag]               3.2.14                                                 -> 3.2.14
 * [new tag]               3.2.15                                                 -> 3.2.15
 * [new tag]               3.2.16                                                 -> 3.2.16
 * [new tag]               3.2.17                                                 -> 3.2.17
 * [new tag]               3.2.2                                                  -> 3.2.2
 * [new tag]               3.2.3                                                  -> 3.2.3
 * [new tag]               3.2.4                                                  -> 3.2.4
 * [new tag]               3.2.5                                                  -> 3.2.5
 * [new tag]               3.2.6                                                  -> 3.2.6
 * [new tag]               3.2.7                                                  -> 3.2.7
 * [new tag]               3.2.8                                                  -> 3.2.8
 * [new tag]               3.2.9                                                  -> 3.2.9
 * [new tag]               3.3.0                                                  -> 3.3.0
 * [new tag]               3.3.1                                                  -> 3.3.1
 * [new tag]               3.3.10                                                 -> 3.3.10
 * [new tag]               3.3.11                                                 -> 3.3.11
 * [new tag]               3.3.12                                                 -> 3.3.12
 * [new tag]               3.3.13                                                 -> 3.3.13
 * [new tag]               3.3.14                                                 -> 3.3.14
 * [new tag]               3.3.15                                                 -> 3.3.15
 * [new tag]               3.3.16                                                 -> 3.3.16
 * [new tag]               3.3.2                                                  -> 3.3.2
 * [new tag]               3.3.3                                                  -> 3.3.3
 * [new tag]               3.3.4                                                  -> 3.3.4
 * [new tag]               3.3.5                                                  -> 3.3.5
 * [new tag]               3.3.6                                                  -> 3.3.6
 * [new tag]               3.3.7                                                  -> 3.3.7
 * [new tag]               3.3.8                                                  -> 3.3.8
 * [new tag]               3.3.9                                                  -> 3.3.9
 * [new tag]               3.4.0                                                  -> 3.4.0
 * [new tag]               3.4.1                                                  -> 3.4.1
 * [new tag]               3.4.10                                                 -> 3.4.10
 * [new tag]               3.4.11                                                 -> 3.4.11
 * [new tag]               3.4.2                                                  -> 3.4.2
 * [new tag]               3.4.3                                                  -> 3.4.3
 * [new tag]               3.4.4                                                  -> 3.4.4
 * [new tag]               3.4.5                                                  -> 3.4.5
 * [new tag]               3.4.6                                                  -> 3.4.6
 * [new tag]               3.4.7                                                  -> 3.4.7
 * [new tag]               3.4.8                                                  -> 3.4.8
 * [new tag]               3.4.9                                                  -> 3.4.9
 * [new tag]               3.5.0                                                  -> 3.5.0
 * [new tag]               3.5.1                                                  -> 3.5.1
 * [new tag]               3.5.10                                                 -> 3.5.10
 * [new tag]               3.5.2                                                  -> 3.5.2
 * [new tag]               3.5.3                                                  -> 3.5.3
 * [new tag]               3.5.4                                                  -> 3.5.4
 * [new tag]               3.5.5                                                  -> 3.5.5
 * [new tag]               3.5.6                                                  -> 3.5.6
 * [new tag]               3.5.7                                                  -> 3.5.7
 * [new tag]               3.5.8                                                  -> 3.5.8
 * [new tag]               3.5.9                                                  -> 3.5.9
 * [new tag]               3.6.0                                                  -> 3.6.0
 * [new tag]               3.6.1                                                  -> 3.6.1
 * [new tag]               3.6.10                                                 -> 3.6.10
 * [new tag]               3.6.11                                                 -> 3.6.11
 * [new tag]               3.6.12                                                 -> 3.6.12
 * [new tag]               3.6.13                                                 -> 3.6.13
 * [new tag]               3.6.14                                                 -> 3.6.14
 * [new tag]               3.6.15                                                 -> 3.6.15
 * [new tag]               3.6.16                                                 -> 3.6.16
 * [new tag]               3.6.17                                                 -> 3.6.17
 * [new tag]               3.6.18                                                 -> 3.6.18
 * [new tag]               3.6.19                                                 -> 3.6.19
 * [new tag]               3.6.2                                                  -> 3.6.2
 * [new tag]               3.6.20                                                 -> 3.6.20
 * [new tag]               3.6.21                                                 -> 3.6.21
 * [new tag]               3.6.3                                                  -> 3.6.3
 * [new tag]               3.6.4                                                  -> 3.6.4
 * [new tag]               3.6.5                                                  -> 3.6.5
 * [new tag]               3.6.6                                                  -> 3.6.6
 * [new tag]               3.6.7                                                  -> 3.6.7
 * [new tag]               3.6.8                                                  -> 3.6.8
 * [new tag]               3.6.9                                                  -> 3.6.9
 * [new tag]               4.0.0                                                  -> 4.0.0
 * [new tag]               4.0.1                                                  -> 4.0.1
 * [new tag]               4.0.10                                                 -> 4.0.10
 * [new tag]               4.0.11                                                 -> 4.0.11
 * [new tag]               4.0.12                                                 -> 4.0.12
 * [new tag]               4.0.13                                                 -> 4.0.13
 * [new tag]               4.0.14                                                 -> 4.0.14
 * [new tag]               4.0.15                                                 -> 4.0.15
 * [new tag]               4.0.16                                                 -> 4.0.16
 * [new tag]               4.0.17                                                 -> 4.0.17
 * [new tag]               4.0.18                                                 -> 4.0.18
 * [new tag]               4.0.19                                                 -> 4.0.19
 * [new tag]               4.0.2                                                  -> 4.0.2
 * [new tag]               4.0.20                                                 -> 4.0.20
 * [new tag]               4.0.21                                                 -> 4.0.21
 * [new tag]               4.0.22                                                 -> 4.0.22
 * [new tag]               4.0.23                                                 -> 4.0.23
 * [new tag]               4.0.24                                                 -> 4.0.24
 * [new tag]               4.0.25                                                 -> 4.0.25
 * [new tag]               4.0.26                                                 -> 4.0.26
 * [new tag]               4.0.27                                                 -> 4.0.27
 * [new tag]               4.0.28                                                 -> 4.0.28
 * [new tag]               4.0.3                                                  -> 4.0.3
 * [new tag]               4.0.4                                                  -> 4.0.4
 * [new tag]               4.0.5                                                  -> 4.0.5
 * [new tag]               4.0.6                                                  -> 4.0.6
 * [new tag]               4.0.7                                                  -> 4.0.7
 * [new tag]               4.0.8                                                  -> 4.0.8
 * [new tag]               4.0.9                                                  -> 4.0.9
 * [new tag]               4.1.0                                                  -> 4.1.0
 * [new tag]               4.1.1                                                  -> 4.1.1
 * [new tag]               4.1.10                                                 -> 4.1.10
 * [new tag]               4.1.11                                                 -> 4.1.11
 * [new tag]               4.1.12                                                 -> 4.1.12
 * [new tag]               4.1.13                                                 -> 4.1.13
 * [new tag]               4.1.14                                                 -> 4.1.14
 * [new tag]               4.1.15                                                 -> 4.1.15
 * [new tag]               4.1.16                                                 -> 4.1.16
 * [new tag]               4.1.17                                                 -> 4.1.17
 * [new tag]               4.1.18                                                 -> 4.1.18
 * [new tag]               4.1.19                                                 -> 4.1.19
 * [new tag]               4.1.2                                                  -> 4.1.2
 * [new tag]               4.1.20                                                 -> 4.1.20
 * [new tag]               4.1.21                                                 -> 4.1.21
 * [new tag]               4.1.22                                                 -> 4.1.22
 * [new tag]               4.1.23                                                 -> 4.1.23
 * [new tag]               4.1.24                                                 -> 4.1.24
 * [new tag]               4.1.25                                                 -> 4.1.25
 * [new tag]               4.1.3                                                  -> 4.1.3
 * [new tag]               4.1.4                                                  -> 4.1.4
 * [new tag]               4.1.5                                                  -> 4.1.5
 * [new tag]               4.1.6                                                  -> 4.1.6
 * [new tag]               4.1.7                                                  -> 4.1.7
 * [new tag]               4.1.8                                                  -> 4.1.8
 * [new tag]               4.2.0                                                  -> 4.2.0
 * [new tag]               4.2.1                                                  -> 4.2.1
 * [new tag]               4.2.10                                                 -> 4.2.10
 * [new tag]               4.2.11                                                 -> 4.2.11
 * [new tag]               4.2.12                                                 -> 4.2.12
 * [new tag]               4.2.13                                                 -> 4.2.13
 * [new tag]               4.2.15                                                 -> 4.2.15
 * [new tag]               4.2.16                                                 -> 4.2.16
 * [new tag]               4.2.17                                                 -> 4.2.17
 * [new tag]               4.2.18                                                 -> 4.2.18
 * [new tag]               4.2.19                                                 -> 4.2.19
 * [new tag]               4.2.2                                                  -> 4.2.2
 * [new tag]               4.2.20                                                 -> 4.2.20
 * [new tag]               4.2.3                                                  -> 4.2.3
 * [new tag]               4.2.4                                                  -> 4.2.4
 * [new tag]               4.2.5                                                  -> 4.2.5
 * [new tag]               4.2.6                                                  -> 4.2.6
 * [new tag]               4.2.7                                                  -> 4.2.7
 * [new tag]               4.2.8                                                  -> 4.2.8
 * [new tag]               4.2.9                                                  -> 4.2.9
From https://github.com/Homebrew/brew
 * [new tag]               4.0.29     -> 4.0.29
 * [new tag]               4.1.9      -> 4.1.9
 * [new tag]               4.2.14     -> 4.2.14
Switched to a new branch 'stable'
==> Updating Homebrew...
==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
################################################################################################################################################################# 100.0%

Error: Checksum mismatch.
Expected: 02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
  Actual: 709a870e862579355398c0115ec4d9995a48daf463ca066d671ca4f40afd9927
 Archive: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
To retry an incomplete download, remove the file above.
Error: Failed to install Homebrew Portable Ruby (and your system version is too old)!
Failed during: /usr/local/bin/brew update --force --quiet

**Comment by gromgit** [comment]
Just to be sure, what's the output of `sw_vers`?

**Reply by elstalk** [reply]
ProductName:	macOS
ProductVersion:	14.4.1
BuildVersion:		23E224

![Screenshot 2024-04-30 at 6 07 52 AM](https://github.com/Homebrew/discussions/assets/35581869/a3cb8b3d-5f26-4aba-a006-01910765736e)

**Reply by elstalk** [reply]
To throw another wrench in, I have homebrew on an M3 MBPro and am having no problems there - which made me wonder if it was something with the Intel chip.

![Screenshot 2024-04-30 at 6 11 25 AM](https://github.com/Homebrew/discussions/assets/35581869/2222a7c3-2c77-4ad6-a5c6-2e4ff182063b)

**Reply by gromgit** [reply]
OK, was wondering about that "your system version is too old" message.

>```
> ==> Downloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
> Already downloaded: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
> Error: Checksum mismatch.
> Expected: 02180ca8b8295422ae84921bcf034b7ee8ce5575488bd5e6a37a192e53cd5d34
>   Actual: 709a870e862579355398c0115ec4d9995a48daf463ca066d671ca4f40afd9927
>  Archive: /Users/elsmith/Library/Caches/Homebrew/portable-ruby-3.1.4.el_capitan.bottle.tar.gz
> To retry an incomplete download, remove the file above.
>```

I've manually downloaded the above, and verified that the expected checksum is indeed correct. Remove the mentioned bottle and try the install again.

**Reply by elstalk** [reply]
Yes, I've tried that, as well. Still gives the error. I believe the "your system version is too old" is referring to the built-in ruby version.

![Screenshot 2024-04-30 at 10 24 17 AM](https://github.com/Homebrew/discussions/assets/35581869/ebfd0329-b3ae-4b66-b2be-5f3cdbc11077)

I'm starting to think I will need to rebuild this machine - I thought maybe y'all would find a problem but it might just be me!

**Reply by gromgit** [reply]
Are you using a corporate web proxy, anti-virus, or some other systems management suite that may be messing with your downloads?

**Reply by elstalk** [reply]
OK I give up... I didn't change anything at all on the machine and now it's working. I'm sorry! Perhaps something changed on our network somewhere!

Thank you so much for your time in responding to me!
